guides/doc-Upgrading_and_Updating/topics/upgrading_a_connected_satellite_server.adoc

```diff
@@ -68,39 +74,70 @@ If you lose connection to the command shell where the upgrade command is running
 ----
 # {foreman-maintain} upgrade run --target-version {TargetVersionMaintainUpgrade}
 ----
-
+endif::[]
+ifdef::katello[]
+. Check for running tasks
++
+[options="nowrap" subs="attributes"]
+----
+# foreman-rake katello:upgrade_check
+----
+. Update operating system packages
++
+[options="nowrap" subs="attributes"]
+----
+# yum update -y
+----
+. Update repositories
++
+.For Centos 7 or Red Hat Enterprise Linux Users:
+[options="nowrap" subs="attributes"]
+----
+# yum update -y https://fedorapeople.org/groups/katello/releases/yum/{KatelloVersion}/katello/el7/x86_64/katello-repos-latest.rpm
+# yum update -y https://yum.theforeman.org/releases/{ProjectVersion}/el7/x86_64/foreman-release.rpm
```

Comment:
I would like to suggest to swap the order: first `foreman-release.rpm` and then update katello. Mostly for a future simplification: I think that we should make a `foreman-release-katello` in the Foreman repository. Then the installation instructions could become:
```
yum install -y https://yum.theforeman.org/releases/{ProjectVersion}/el7/x86_64/foreman-release.rpm
yum install -y foreman-release-katello centos-release-scl-rh
```
(name could also be `katello-release`, not important for the example)

The benefit is that there is no (reasonable) way for a user to install a mismatched version.